mosec/server.py

```diff
@@ -97,16 +99,18 @@ def _terminate(self, signum, framestack):
     @staticmethod
     def _clean_pools(
         processes: List[Union[mp.Process, None]],
-        events: List[Union[Event, None]],
-    ) -> Tuple[List[Union[mp.Process, None]], List[Union[Event, None]]]:
-        for i, (p, e) in enumerate(zip(processes, events)):
+    ) -> List[Union[mp.Process, None]]:
+        for i, p in enumerate(processes):
             if p is None or p.exitcode is not None:
                 processes[i] = None
-                events[i] = None
-        return processes, events
+        return processes
 
     def _manage_coordinators(self):
         first = True
+        shutdown = mp.Event()
+        shutdown_notify = mp.Event()
+        self._coordinator_shutdown = shutdown
+        self._coordinator_shutdown_notify = shutdown_notify
```

Comment:
No need to create two local variables. `self._xxx` can explicit indicate that it's using the same instance variables.